templates/configmap.yaml

```diff
@@ -75,6 +75,7 @@ data:
     worker_container_image_pull_policy = {{ .Values.images.airflow.pullPolicy }}
     dags_in_image = True
     delete_worker_pods = True
+    delete_on_success = False
```

Comment:
is this needed, or just leftover from your debugging?

Reply:
@kaxil I think in general this is the best setting for debugging (basically only keeps pods up if they fail), though I'm willing to hear otherwise if that's not ideal for cloud

Reply:
yea I fear for cloud customers, as they can't delete the pods, this will increase their usage.

cc @schnie

Reply:
I have removed it from this PR to unblock merging of this PR, feel to create a new PR to add that